implementations/rust/ockam/ockam_transport_udp/src/transport/lifecycle.rs

```diff
@@ -20,15 +21,30 @@ impl UdpTransport {
     /// # Ok(()) }
     /// ```
     #[instrument(name = "create udp transport", skip_all)]
-    pub fn create(ctx: &Context) -> Result<Self> {
-        let udp = Self {
-            ctx: Arc::new(ctx.try_clone()?),
-        };
-        // make the UDP transport available in the list of supported transports for
-        // later address resolution when socket addresses will need to be instantiated as UDP
-        // worker addresses
-        ctx.register_transport(Arc::new(udp.clone()));
-        Ok(udp)
+    pub fn create(ctx: &Context) -> Result<Arc<UdpTransport>> {
```

Comment:
Shouldn't this be named `get_or_create` ?

Reply:
Yes, much better.